fhir/com.b2international.snowowl.fhir.core/src/org/linuxforhealth/fhir/model/annotation/Summary.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.linuxforhealth.fhir.model.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSortedSet;
+
+/*
+ * Modifications:
+ * 
+ * - Extended annotation with constants holding possible values
+ *   (see http://hl7.org/fhir/R5/search.html#_summary)
+ */
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Summary {
```

Comment:
Are any of these annotations processed somewhere or they are mainly for documentation purposes?

Reply:
I'd like to slowly work these external domain objects into the requests, making the conversion step unneccessary. We are using this annotation in our current model as well &ndash; when the query parameter `_summary` is added to the request, only properties marked with &commat;`Summary` are appearing in the response.